notifications.json

```diff
@@ -0,0 +1,861 @@
+                    "properties": {
+                        "icon": {
+                            "type": "string"
+                        },
+                        "sound": {
+                            "type": "string"
+                        },
+                        "color": {
+                            "type": "string"
+                        },
+                        "badge": {
+                            "type": "string"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "title"
+            ]
+        },
+        "message-input": {
+            "title": "Message Input",
+            "type": "object",
+            "properties": {
+                "to": {
+                    "description": "One fiscal code (CF).\n",
+                    "type": "string"
+                },
+                "from": {
+                    "type": "string"
+                },
+                "client_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "reference_id": {
+                    "type": "string",
+                    "description": "An optional identifier you generate if you don’t want to use notify’s id. It can be used to identify a single notification or a batch of notifications."
+                },
+                "reply_to_message_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "thread_id": {
+                    "type": "string",
+                    "format": "uuid"
+                },
+                "priority": {
+                    "type": "integer",
+                    "minimum": 0,
+                    "maximum": 10
+                },
+                "dry_run": {
+                    "type": "boolean",
+                    "description": "This parameter, when set to true, allows developers to test a request without actually sending a message.\n\nThe default value is false."
+                },
+                "time_to_live": {
+                    "type": "integer",
+                    "description": "This parameter specifies how long (in seconds) the message should be kept in storage if the device is offline. The maximum time to live supported is 4 weeks, and the default value is 4 weeks.",
+                    "minimum": 0
+                },
+                "expires_at": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "tracking": {
```

Comment:
to turn on tracking (ie. link tracking on emails or number of opened emails and so on)